docs/src/installation_instructions.md

```diff
@@ -6,15 +6,16 @@ Oceananigans. If you're new to Julia, we also recommend reading [Julia's "gettin
 
 ## Installation instructions
 
-The hardest part of installing Oceananigans is actually installing Julia (and even that should be pretty easy). 
-If you want to use Oceananigans on a laptop you can generally install Julia easily via the package manager of whatever
-operational system you use (some quick googling should tell you how). If you're running on a cluster, you should check
-if it already has Julia installed (most systems these days do). If that's the case, then the hardest part is already done!
-
-If those options above are not possible, one alternative is to simply download the Julia pre-compiled binaries from the
-[Julia website](https://julialang.org/downloads/), where you can simply run the Julia executable that's contained inside
-(this should work out of the box!). Another option is to compile Julia manually from source code. This is definitely the hardest
-option, and you can find instructions on how to do that [here](https://github.com/JuliaLang/julia#building-julia).
+Usually the hardest part of installing Oceananigans is installing Julia. The main options here in order
+of simple to complex are
+
+1. Check if your system already has Julia installed. Most HPC systems these days have Julia
```

Comment:
Should we add a sentence like?: 

If you are using an HPC, run `module spider julia` to check the available versions. For now, `Oceananigans` is only supported by `Julia-1.6.5 (LTS)`.

Reply:
I don't know if `module spider` is common to all HPC systems, so I'm reluctant to include that info there

Reply:
I also don't know. haha
Well, the clusters I have used so far have different jobs managers and still works with `module avail` or `module spider`.
What do you think @glwagner and @navidcy ?

Reply:
I think its not right for the docs which should focus on documenting the code. I think it'd be great to have helpful info and tips like that somewhere for users. A discussion might be a great place because everyone can contribute and its more living (we don't need PRs to update it)

Reply:
I agree with @glwagner. The docs should make clear which version of Julia Oceananigans.jl require (which they do). We cannot document in Oceananigans docs details about all HPC systems.